File: __tests__/__fixtures__/output/java/asynchttp/multipart-form-data-no-params.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("POST", "https://httpbin.org/anything")
  .setHeader("Content-Type", "multipart/form-data")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();

File: __tests__/__fixtures__/output/node/unirest/multipart-form-data-no-params.js
const unirest = require("unirest");

const req = unirest("POST", "https://httpbin.org/anything");

req.headers({
  "Content-Type": "multipart/form-data"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

